### ISSUE TITLE
Version Bump Request

### ISSUE THREAD

**Issue by Jitsusama** [brycied00d/fluent-plugin-splunk-http-eventcollector#29]
Hello,

First of all, thanks for maintaining this plugin! I'm using and loving it!!

That said, there has been some new features added that I would really like to use, but without having a version bump, I can't `gem install` them. In particular, the microsecond accuracy fix is a critical feature for my company.

Would you mind finding the time to release a new version with this (and any other changes you are comfortable with) included?

Thanks!

**Comment by ntrp** [comment]
Yeah would be amazing! At the moment I am patching it during automation..
